tidymodels-regression-part2.qmd

```diff
@@ -0,0 +1,274 @@
+# Regression in Tidymodels - Part 2
+
+This chapter is starts off as the Regression in Tidymodels chapter - Epidemiological time series edition and goes from there to making (slightly) more sophisticated predictions and interactive plots.
+
+```{r}
+#| echo: false
+source("_common.R")
+```
+
+## Libraries and data
+
+```{r, message = FALSE}
+# Load necessary packages
+library(tidymodels)
+library(plotly)
+```
+
+As a follow-up to the Regression in `Tidymodels` chapter, we're going to look at how the functions we used to perform linear regression fair when applied to a small set of epidemiological time series data. We'll be working with the built-in data on daily confirmed COVID-19 case and death rates for California over Dec. 31, 2020 to Dec. 31, 2021 (this was originally compiled by JHU).
```

Comment:
Should link to the previous chapter. See: https://quarto.org/docs/books/book-crossrefs.html

Reply:
I think "fare" not "fair".